### DISCUSSION TITLE
Runway Events?

### DISCUSSION THREAD

**Discussion by IVoyt** [statamic-rad-pack/runway#520] [Help] [answered]
### Description

Does Runway support events? The Statamic's built-in events do not fit my task since Runway's models are not Collections and Entries in context of Statamic therefore I can't listen saving events of Runway models. Is there a way to do that?

### Environment

Environment
Application Name: LIGA CMS
Laravel Version: 11.8.0
PHP Version: 8.2.13
Composer Version: 2.2.7
Environment: local
Debug Mode: ENABLED
URL: site.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: sqlite
Logs: stack / single
Mail: log
Queue: sync
Session: file

Statamic
Addons: 6
Sites: 2 (Test Site 1, Test Site 2)
Stache Watcher: Enabled
Static Caching: Disabled
Version: 5.7.2 PRO

Statamic Addons
eminos/statamic-tabs: 1.2.0
mitydigital/statamic-tinymce-cloud: 3.0.0
statamic-rad-pack/runway: 7.3.1
statamic/collaboration: 1.0.0
statamic/eloquent-driver: 4.0.0
statamic/ssg: 3.0.1

Statamic Eloquent Driver
Asset Containers: eloquent
Assets: eloquent
Blueprints: file
Collection Trees: eloquent
Collections: eloquent
Entries: eloquent
Forms: eloquent
Global Sets: eloquent
Global Variables: eloquent
Navigation Trees: eloquent
Navigations: eloquent
Revisions: file
Taxonomies: eloquent
Terms: eloquent

**Comment by duncanmcclean** [comment, marked as answer]
Runway doesn’t provide any of its own events. However, you’re able to take advantage of [Eloquent’s model events](https://laravel.com/docs/11.x/eloquent).
